src/gnatss/posfilter/posfilter.py

```diff
@@ -29,6 +29,8 @@ def spline_interpolate(
         Pandas Dataframe containing Antenna enu directions std deviation Novatel L1 data
     twtt_df : pd.DataFrame
         The travel times data
+    full_result : bool
+        TODO: Add description

```

Comment:
Add something along the lines of

"Flag to either export the full RPH data from the instrument or just the data that aligns with travel times"